tensorflow/lite/micro/kernels/xtensa/quantize.cc

```diff
@@ -135,6 +151,17 @@ TfLiteStatus EvalXtensa(TfLiteContext* context, TfLiteNode* node) {
       switch (output->type) {
         case kTfLiteInt8: {
           int size = ElementCount(*input->dims);
+#if defined(HIFI5) || defined(HIFI4) || defined(HIFI4_INTERNAL)
+          TF_LITE_ENSURE_EQ(
+              context,
+              xa_nn_elm_requantize_asym16s_asym8s(
+                  tflite::micro::GetTensorData<int8_t>(output),
+                  tflite::micro::GetTensorData<int16_t>(input),
+                  op_data->input_zero_point, op_data->quantization_params.zero_point,
+                  op_data->requantize_output_shift,
+                  op_data->requantize_output_multiplier, size),
+              0);
+#else // defined(HIFI5) || defined(HIFI4) || defined(HIFI4_INTERNAL)          
```

Comment:
This #else codeblock is the same as the #if. It seems like this code could have just been left as is?

Reply:
else block should fall back to reference op

Reply:
Actually, looks like none of these #ifdefs are needed. In the Eval function, we already do a check for HIFI3, 4, & 5.  If those aren't defined, we'll use a reference implementation of Eval. I think the only #ifdef check still needed is the HIFI_VFPU for the float quantizations. I'll add a commit to this effect. Please let me know if I get anything wrong.